covid-projections/act-now-packages
# Create validation package

Implements a validation package to replace some of our `assert` and `fail` usages.

Part of https://github.com/act-now-coalition/act-now-packages/issues/561

To test:
1. Checkout this PR and run storybook.
2. In `MetricLegendCategorical`, change the metric of any of the stories to `MetricId.PI`. The story should throw an `ActNowJsError` with the appropriate error message.
